docs/pages/router/reference/search-parameters.mdx

```diff
@@ -0,0 +1,200 @@
+---
+title: Using Search Parameters
+description: Learn how to access and modify search parameters in your app.
+hideFromSearch: true
+sidebar_title: Search Parameters
+---
+
+import { FileTree } from '~/ui/components/FileTree';
+import { Terminal } from '~/ui/components/Snippet';
+import { Step } from '~/ui/components/Step';
+import { Tabs, Tab } from '~/ui/components/Tabs';
+
+Search parameters (also known as query params) are serializable fields that can be added to a URL. They are commonly used to pass data between pages.
+
+Expo Router provides hooks for accessing and modifying search parameters.
+
+In nested apps, you'll often have multiple "pages" mounted at a time, for example a stack has the previous page and current page in memory when a new route is pushed. Because of this, Expo Router provides two different hooks for accessing search parameters:
+
+- **useLocalSearchParams**: returns the search parameters for the current component. This only updates when the global URL conforms to the route.
+- **useGlobalSearchParams**: returns the global URL regardless of the component. This updates on every search param change and might cause components to update extaneously in the background.
+
+Both hooks are typed and accessed the same way, the only difference is how frequently they update. Consider the following example:
+
+## Local vs. Global Search Parameters
+
+The following example demonstrates the difference between `useLocalSearchParams` and `useGlobalSearchParams`.
+
+<FileTree files={['app/_layout.js', 'app/index.js', 'app/[user].js']} />
+
+<Step label="1">
+
+    The Root Layout will be a stack navigator.
```

Comment:
```suggestion
    The Root Layout is a stack navigator:
```

Reply:
[Using present tense for our audience as per writing styleguide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#use-present-tense)